.github/workflows/test.yaml

```diff
@@ -130,7 +130,7 @@ jobs:
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-24.06
     with:
-      matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
+      matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | min(max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
```

Comment:
We have the same job in `pr.yaml` and here. One is run on nightly and one on pr's, both have different matrix combinations: https://github.com/rapidsai/shared-workflows/blob/13e008c746e30a830c4ebe83a6786862858dfcb8/.github/workflows/wheels-test.yaml#L75-L94

To achieve a consistent matrix for comparison, I changed this to `min` so that we compare with `cudf-12.2.2` and python 3.9 versions for both jobs.

Reply:
This is a coincidence that we have the same minimum. Can we find another way to get a common configuration, like using the “latest” image?

Reply:
Do you think it's okay to hard-code here instead of `map(select..`? Again, can we do this as a follow-up?